Sources/StreamChatUI/ChatChannelList/ChatChannelListItemView.swift

```diff
@@ -102,30 +102,38 @@ open class ChatChannelListItemView: _View, ThemeProvider, SwiftUIRepresentable {
         guard let content = content else { return nil }
         if let typingUsersInfo = typingUserString {
             return typingUsersInfo
-        } else if let latestMessage = content.channel.latestMessages.first {
-            return "\(latestMessage.author.name ?? latestMessage.author.id): \(latestMessage.textContent ?? latestMessage.text)"
+        } else if let previewMessage = content.channel.previewMessage {
```

Comment:
Probably this is already a performance improvement since instead of fetching all the latest messages, we now directly fetch the preview message. 👍 Would be nice if this issue: CIS-1677 (https://github.com/GetStream/stream-chat-swift/issues/1805) is still reproducible

Reply:
I'll run the load test and check if I it's better now 👍

Reply:
The issue is not fully resolved because we still do a lot on the main thread but it's much better 👍Previously, the weight of `subtitle` calculation was 3.8% from everything we do on main thread, with `previewMessage` it's `0.1` 🎉

Reply:
The weight of `updateContent` has increased because we're now showing/hiding/moving around the delivery checkmark but that's unavoidable when adding the new subview